garden/src/components/Navbar.tsx

```diff
@@ -4,15 +4,16 @@ import { Link } from "react-router-dom";
 const Navbar = () => {
   return (
     <nav className="border-b border-gray-200 flex justify-between px-8 py-2">
+      <Link to="/home">
       <img
         src="img/smallColorIcon_Garden.png"
         alt="Garden AI Logo"
         className="w-28 object-contain"
       ></img>
+      </Link>
       <ul className="flex items-center gap-8">
         <Link to="/home">Home</Link>
-        {/* <Link to="/search">Search</Link> */}
```

Comment:
Is the search link supposed to be commented out? It looks like the about link is still there too?

Reply:
I think that is just showing that they were removed, which is why they are in red? The search link is uncommented and the about link is gone when I look at the code.

Reply:
oooooh sorry, Chase! My bad. I tried to look really quickly and that's what I get. Merging in now!